lib/aganakti.rb

```diff
@@ -111,17 +111,22 @@ def new(uri, options = {}) # rubocop:disable Metrics/AbcSize, Metrics/Cyclomatic
       Ethon::Curl.version
     ].compact.join(' ')
 
+    # Form the options for Aganakti::Client to pass to Typhoeus
+    client_options = {
+      accept_encoding: '', # allow libcurl to determine its supported compression
+      headers:         {
+        'Connection' => 'keep-alive',
+        'User-Agent' => user_agent
+      },
+      tcp_keeppalive:  true
+    }
+
+    client_options[:connecttimeout] = options[:connect_timeout] if options.key?(:connect_timeout)
+    client_options[:cainfo] = options[:tls_ca_certificate_bundle] if options.key?(:tls_ca_certificate_bundle)
+    client_options[:timeout] = options[:timeout] if options.key?(:timeout)
```

Comment:
```suggestion
    client_options[:connecttimeout] = options[:connect_timeout] if options.key?(:connect_timeout)
    client_options[:cainfo]         = options[:tls_ca_certificate_bundle] if options.key?(:tls_ca_certificate_bundle)
    client_options[:timeout]        = options[:timeout] if options.key?(:timeout)
```

but this is really being pedantic